.travis.yml

```diff
@@ -0,0 +1,95 @@
+language: cpp
+cache: ccache
+os:
+  - linux
+  - osx
+dist: xenial
+osx_image: xcode10.1
+compiler:
+  - clang
+  - gcc
+addons:
+  apt:
+    packages:
+      - cmake
+      - doxygen
+      - libpng-dev
+      - libxml2-utils
+      - llvm
+      - clang
+      - docutils-common
+      - libomp-dev
+      - lcov
+  homebrew:
+    packages:
+      - doxygen
+      - glib
+      - libpng
+      - libxml2
+      - ccache
+      - libomp
+notifications:
+  webhooks:
+    - https://coveralls.io/webhook
+env:
+  global:
+    - COVERALLS_PARALLEL=true
+  matrix:
+    - build_type=release
+    - build_type=release CMAKE_EFLAGS="-DBUILD_STATIC=ON -DBUILD_LENSTOOL=ON"
+    - build_type=debug
+    - build_type=debug CMAKE_EFLAGS="-DBUILD_STATIC=ON -DBUILD_LENSTOOL=ON"
+    - build_type=Coverage
```

Comment:
This is weird. Why is build_type overloaded with whether to collect coverage info or not? This goes into CMake as well as CMAKE_BUILD_TYPE which doesn't really make sense.

Reply:
Someone set up the cmakelists to set up coverage is build type equals coverage

Reply:
I see, OK, no problem then.